src/domUpdates.js

```diff
@@ -0,0 +1,73 @@
+let domUpdates = {
```

Comment:
Awesome! We were really really close to getting things going on Saturday. Thanks for all your hard work.